Makefile

```diff
@@ -20,6 +20,7 @@ FL2M_BIN_FILE=flowlogs2metrics
 CG_BIN_FILE=confgenerator
 NETFLOW_GENERATOR=nflow-generator
 CMD_DIR=./cmd/
+FL2M_CONF_FILE ?= contrib/kubernetes/flowlogs2metrics.conf.yaml
```

Comment:
using this `FL2M_CONF_FILE` as the name for the configuration file: This is used twice in the makefile + this allows us to deploy with different file than the default.